README.md

```diff
@@ -133,23 +133,59 @@ docker-compose -f docker-compose.test.yml up --build --exit-code-from test
 ~~~
 
 ### Developing / Debugging
-Build and start your container in "developer mode"
+
+1.  Build your container in "developer mode":
 ~~~bash
-./scripts/devel-compose build --no-cache vmaas_webapp
-./scripts/devel-compose up vmaas_webapp
+./scripts/devel-compose build --no-cache
+./scripts/devel-compose up
 ~~~
 
-switch inside of the container
+2.  Switch into **webapp** container:
+
 ~~~bash
-./scripts/devel-compose exec vmaas_webapp bash
+docker-compose exec vmaas_webapp bash
 ~~~
 
-now your local git directory is mounted under `/git` in the container so any change
+Now your local git directory is mounted under `/git` in the container so any change
 you make you can immediatelly test.
 
-```[root@4cb6b50d0cb6 git]# python3 ./app.py```
+```[root@localhost git]# python3 ./app.py```
```

Comment:
I would remove all `[root@localhost git]#` prompts and write only bash code. It complicates copy-paste process. You can also you bash notation and then use standard bash comments if you need:
~~~bash
python3 ./app.py # this is called in git repo
~~~